ManatiCR/trichechus
# AS-13: Update text editors

# [AS-13](https://manati.atlassian.net/browse/AS-13)

- Added the **Full Editor** editor.
- Update the **Full HTML** editor to be restricted of html tags.
- Deleted the `video_embed_field` module.
- Added `media_entity_instagram` and `url_embed` modules.
- Added new **Instagram** Media entity.
- Added `view modes` for the **Image** Media Entity to be used in the ckeditor.

# Steps to test
- [ ] Clone project and switch to branch
- [x] Move the cloned folder to the profiles folder
- [ ] Add all the needed config listed in the [README](https://github.com/ManatiCR/trichechus/blob/feature/update-profle-config/README.md) file
- [ ] Install the Drupal site using this profile
- [ ] Go to `/admin/config/content/formats` and confirm the `Full Editor` is already there and it's configured enough.
- [ ] Go to `admin/structure/media` and confirm the following:
    - The Instagram media type exist and it's configured.
    - The Image media type has as many display modes as site image styles.
    - All the Media entities have description.
- [ ] Add a new body field within `Landing page` Content type, and checkout the text formats work right:
    - Using both `Full HTML` or `Full editor`  add an image entity and align it to right or center. Save the content and confirm the class is added to the final HTML element.
     - Using both `Full HTML` or `Full editor`  add an Instagram media entity and confirm it's displayed right.
     - Using `Full editor` add an URL embed from wherever you want and confirm it's displayed right.
